test/model/content_test.dart

```diff
@@ -395,6 +395,23 @@ class ContentExample {
       ParagraphNode(links: null, nodes: [TextNode("some content")]),
     ]);
 
+  static const codeBlockNodesWithMultipleClasses = ContentExample(
```

Comment:
nit: put this just after the two `codeBlockHighlighted…` examples, so it's not separated from them by all the unimplemented code-block examples